Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM fundocker/edxapp:hawthorn.1-2.0.1
+
+ARG UID=1000
+ARG GID=1000
+
+# Switch back to a priviledged user to perform base installation
+USER root:root
+
+# Add the non-privileged user that will run the application
+RUN groupadd --gid ${GID} edx && \
+    useradd --uid ${UID} --gid ${GID} --home /edx edx
+
+# Allow the edx user to create files in /edx/var (required to perform database
+# migrations)
+RUN mkdir /edx/var && \
+    chown edx:edx /edx/var
+
+# Override LMS settings used for development (with xapi enabled)
+COPY ./docker/config/lms/docker_run_development.py /config/lms/
+
+# Copy the app to the working directory
+COPY --chown=edx:edx . /edx/app/xapi/
+
+# Install development dependencies in a virtualenv
+RUN pip install --no-cache-dir --src /usr/local/src -e /edx/app/xapi
+
+# FIXME: as mentionned in fun-platform and edx-platform bug tracker, this
+# webpack-stats.json is required both in production and development in a static
+# directory 😢
+RUN mkdir /edx/app/edxapp/staticfiles && \
+    cp /edx/app/edxapp/edx-platform/common/static/webpack-stats.json /edx/app/edxapp/staticfiles/
```

Comment:
but this is already done in the image you inherit from?

Reply:
Unfortunately no: this file is available in the `builder` and `development` targets, but not in `production` where you are supposed to perform a `collectstatic`.

Reply:
I'm not comfortable with this as the production image is not supposed to have "collected" static files... which will be override _via_ a docker volume in 90% cases. I might be wrong but I think it's best to fix horrible things in derived images than in the production one. WDYT?